### ISSUE TITLE
non determinism when compressing images with default compr level

### ISSUE THREAD

**Issue by matthiaskrgr** [shssoichiro/oxipng#128]
sometimes an image is reduced in size, sometimes it isn't:
image: https://github.com/wesnoth/wesnoth/blob/master/data/core/images/wesnoth-icon.png

````
 matthias@t470  ~/vcs/github/wesnoth/data/core/images   master  md5sum wesnoth-icon.png; oxipng wesnoth-icon.png
bd37ad1c87b104479b10745995615656  wesnoth-icon.png
Processing: wesnoth-icon.png
    128x128 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 25214 bytes
    File size = 25365 bytes
Trying: 8 combinations
Found better combination:
    zc = 9  zs = 3  f = 5        24639 bytes
    IDAT size = 24639 bytes (575 bytes decrease)
    file size = 24754 bytes (611 bytes = 2.41% decrease)
Output: wesnoth-icon.png
 matthias@t470  ~/vcs/github/wesnoth/data/core/images   master ●  md5sum wesnoth-icon.png; oxipng wesnoth-icon.png
8900eb871d360daa6626b51f205a384b  wesnoth-icon.png
Processing: wesnoth-icon.png
    128x128 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 24639 bytes
    File size = 24754 bytes
Trying: 8 combinations
Found better combination:
    zc = 9  zs = 3  f = 5        24638 bytes
    IDAT size = 24638 bytes (1 bytes decrease)
    file size = 24753 bytes (1 bytes = 0.00% decrease)
Output: wesnoth-icon.png
 matthias@t470  ~/vcs/github/wesnoth/data/core/images   master ●  md5sum wesnoth-icon.png; oxipng wesnoth-icon.png
c9b10cff7d4746eee1561980194b74dc  wesnoth-icon.png
Processing: wesnoth-icon.png
    128x128 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 24638 bytes
    File size = 24753 bytes
Trying: 8 combinations
Found better combination:
    zc = 9  zs = 3  f = 5        24637 bytes
    IDAT size = 24637 bytes (1 bytes decrease)
    file size = 24752 bytes (1 bytes = 0.00% decrease)
Output: wesnoth-icon.png
 matthias@t470  ~/vcs/github/wesnoth/data/core/images   master ●  md5sum wesnoth-icon.png; oxipng wesnoth-icon.png
00783470b08846378a8ed780b16a918e  wesnoth-icon.png
Processing: wesnoth-icon.png
    128x128 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 24637 bytes
    File size = 24752 bytes
Trying: 8 combinations
    IDAT size = 24637 bytes (0 bytes decrease)
    file size = 24752 bytes (0 bytes = 0.00% decrease)
File already optimized
 matthias@t470  ~/vcs/github/wesnoth/data/core/images   master ●  md5sum wesnoth-icon.png; oxipng wesnoth-icon.png
00783470b08846378a8ed780b16a918e  wesnoth-icon.png
Processing: wesnoth-icon.png
    128x128 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 24637 bytes
    File size = 24752 bytes
Trying: 8 combinations
    IDAT size = 24637 bytes (0 bytes decrease)
    file size = 24752 bytes (0 bytes = 0.00% decrease)
File already optimized
 matthias@t470  ~/vcs/github/wesnoth/data/core/images   master ●  md5sum wesnoth-icon.png; oxipng wesnoth-icon.png
00783470b08846378a8ed780b16a918e  wesnoth-icon.png
Processing: wesnoth-icon.png
    128x128 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 24637 bytes
    File size = 24752 bytes
Trying: 8 combinations
    IDAT size = 24637 bytes (0 bytes decrease)
    file size = 24752 bytes (0 bytes = 0.00% decrease)
File already optimized
 matthias@t470  ~/vcs/github/wesnoth/data/core/images   master ●  md5sum wesnoth-icon.png; oxipng wesnoth-icon.png
00783470b08846378a8ed780b16a918e  wesnoth-icon.png
Processing: wesnoth-icon.png
    128x128 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 24637 bytes
    File size = 24752 bytes
Trying: 8 combinations
Found better combination:
    zc = 9  zs = 3  f = 5        24629 bytes
    IDAT size = 24629 bytes (8 bytes decrease)
    file size = 24744 bytes (8 bytes = 0.03% decrease)
Output: wesnoth-icon.png
````

Happens with optimized and non-optimized builds of oxipng 2.0.2

**Comment by andrews05** [comment]
This is a really old issue, but I'll just mention that since v2.0.2 some determinism issues have been identified and resolved in #199 and #463. So try again with the current version, it's highly likely the issue no longer occurs.

**Comment by andrews05** [comment]
Closing as presumed fixed. Feel free to reopen if necessary.
